### PR TITLE
className convention fixed

### PR DESCRIPTION
**Pull request by Simpcyclassy**

### CODE REVIEW

File: src/components/signup/components/SignupForm.js [hunk 1]
```diff
@@ -23,25 +23,23 @@ const {
...
-                    decorator(field, {
-                        rules,
-                    })(
-                        <Input type={inputType} />
-                    )
-                }
-            </FormItem>
-        );
-    });
+    return INPUTS.map(({ field, inputType, label, rules }) =>
```
Comment (Way2nnadi) [inline src/components/signup/components/SignupForm.js]: quick learner, nice use of destructuring here.

**Comment by Way2nnadi** [comment]
LGTM

File: src/components/signup/components/SignupForm.js (hunk 1 above)
Comment (Simpcyclassy) [reply src/components/signup/components/SignupForm.js]: Thank you 😊

Comment (EngrEric) [reply src/components/signup/components/SignupForm.js]: neat and concise

**Comment by EngrEric** [comment]
Looks good to me

File: src/components/signup/components/SignupForm.js (hunk 1 above)
Comment (joshuaobinna) [reply src/components/signup/components/SignupForm.js]: well done

**Comment by joshuaobinna** [comment]
looks good to me

**Pull request merged**